### PR TITLE
feat: use new sentry sdk

### PR DESCRIPTION
**Pull request by aldarund**
Sentry now have new packages. While old one will work, there wont be any new features added, only bugfixes.
https://forum.sentry.io/t/switching-to-sentry-javascript/4732

Not tested, help with testing welcome ;)

### CODE REVIEW

**Comment by DiederikvandenB** [comment]
Nice. This will be included in the new v2.0 release, as NuxtJS recently published v2.0 themselves.

After this has been merged, we can close #20 as well.

Are you planning on adding tests yourself?

**Comment by aldarund** [comment]
@DiederikvandenB ye, i`l add something and test this week

**Comment by tillkruss** [comment]
Sweet, looks like a seamless upgrade.

File: lib/templates/sentry-client.js [hunk 1]
```diff
@@ -1,14 +1,16 @@
...
+    integrations: (integrations) => {
+      integrations.push(new Sentry.Integrations.Vue({ Vue }))
+      return integrations
+    }
+  })
+  Sentry.init(opts)
+
+  // Inject Sentry to the context as $sentry
+  ctx.$sentry = Sentry
+  inject('$sentry', Sentry)
```
Comment (seralichtenhahn) [inline lib/templates/sentry-client.js]: The dollar sign `$` in the `inject()` method is not needed because it will be injected by `nuxt` automatically.

**Comment by DiederikvandenB** [comment]
@aldarund, is this ready to be merged in?

The tests would be nice of course, but not completely essential.

**Comment by aldarund** [comment]
@DiederikvandenB i tried to add test, but wasnt able to properly mock sentry for testing purposes, so no test.
Other than that it should be good to go. One point
I removed $raven and now its $sentry ( like in official sentry lib ). In case you want backward compatibility $raven could be added as alias of $sentry.

**Comment by DiederikvandenB** [comment]
I have decided to follow the PR. Will release a breaking change 2.0

**Pull request merged**